ingestion_tools/scripts/gjensen_config.py

```diff
@@ -707,12 +706,9 @@ def handle_per_run_param_maps(
             ret[param_name] = next(iter(values.keys()))
         else:
             key = f"float {{{param_name}}}"
-            for param_value, runs in distinct_values[param_name].items():
-                for run_name in runs:
-                    run_data_map[run_name][param_name] = param_value
             ret[param_name] = key
 
-    return ret, run_data_map
```

Comment:
As we are removing the addition to run_data_map, the  case where there are different values for `frame_dose_rate` for runs in a dataset is not handled correctly.